guides/v2.3/graphql/reference/store-config.md

```diff
@@ -1,38 +1,80 @@
-`locale` | String | The store's locale, such as `en_US`
 `base_currency_code` | String | The code representing the currency in which Magento processes all payment transactions, such as `USD`
-`default_display_currency_code` | String | The code representing the currency displayed on the store, such as `USD`
-`timezone` | String | The store's time zone, such as `America/Chicago`
-`weight_unit` | String | The weight unit for products, such as `lbs` or `kgs`
-`base_url` | String | The store's fully-qualified base URL, such as `http://magentohost.example.com/`
 `base_link_url` | String | A fully-qualified URL that is used to create relative links to the `base_url`
 `base_static_url` | String | The fully-qualified URL that specifies the location of static view files, such as `http://magentohost.example.com/pub/static/`
 `base_media_url` | String | The fully-qualified URL that specifies the location of user media files, such as `http://magentohost.example.com/pub/media/`
-`secure_base_url` | String | The store's fully-qualified secure base URL, such as `https://magentohost.example.com/`
-`secure_base_link_url` | String | A fully-qualified URL that is used to create relative links to the `base_url`
-`secure_base_static_url` | String | The secure fully-qualified URL that specifies the location of static view files, such as `https://magentohost.example.com/pub/static/`
+`base_url` | String | The store's fully-qualified base URL, such as `http://magentohost.example.com/`
+`code` | String | A unique identifier for the store
+`default_display_currency_code` | String | The code representing the currency displayed on the store, such as `USD`
+`id` | Integer | The ID number assigned to the store
+`locale` | String | The store's locale, such as `en_US`
+`secure_base_link_url` | String | A secure fully-qualified URL that is used to create relative links to the `base_url`
 `secure_base_media_url` | String | The secure fully-qualified URL that specifies the location of user media files, such as `https://magentohost.example.com/pub/media/`
+`secure_base_static_url` | String | The secure fully-qualified URL that specifies the location of static view files, such as `https://magentohost.example.com/pub/static/`
+`secure_base_url` | String | The store's fully-qualified secure base URL, such as `https://magentohost.example.com/`
+`timezone` | String | The store's time zone, such as `America/Chicago`
+`website_id` | Integer | The ID number assigned to the parent website  
+`weight_unit` | String | The weight unit for products, such as `lbs` or `kgs`
+{:style="table-layout:auto;"}
+
+### Supported theme attributes
+
+Use the `theme` attributes to retrieve information about the store's thematic elements; such as, footer and header information, copyright text, and logo information.
+
+Attribute |  Data Type | Description
+--- | --- | ---
+`absolute_footer` | String | Contains scripts that must be included in the HTML before the closing &lt;body&gt; tag
```

Comment:
```suggestion
`absolute_footer` | String | Contains scripts that must be included in the HTML before the closing `<body>` tag
```

Reply:
You don't need to escape the less/greater than signs.

Reply:
fixed